static/src/pages/solutions/database.astro

```diff
@@ -104,7 +102,7 @@ const metadata = {
       {
         title: 'Machine Learning',
         description:
-          'Batteries Included provides an integrated development environment to help your team get machine learning projects off the ground faster, and with better results. Prepare data, host models, and manage pipelines with ease. Our environment makes it easy to integrate machine learning projects into your team’s day-to-day, with space for notebooks, code, data, and more.',
+          'Batteries Included provides an easy to use vector databases and more tools ready at the press of a button. Our environment makes it easy to integrate machine learning projects into your team’s day-to-day, with embedding models, jypter notebooks, and PGVector, while keeping your data local and secure.',
```

Comment:
jypter -> Jupyter

Reply:
change:
`Batteries Included provides an easy to use vector databases...`
to:
`Batteries Included provides easy to use vector databases`